storage/persistent_storage/persistent-storage-ocs.adoc

```diff
@@ -23,73 +23,73 @@ OpenShift Container Storage on top of Red Hat Hyperconverged Infrastructure (RHH
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_openshift_container_storage_using_bare_metal_infrastructure/[Deploying OpenShift Container Storage 4.8 using bare metal infrastructure]
 
 |Deploying Red Hat OpenShift Container Storage to use an external Red Hat Ceph Storage cluster
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_openshift_container_storage_in_external_mode/[Deploying OpenShift Container Storage 4.7 in external mode]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_openshift_container_storage_in_external_mode/[Deploying OpenShift Container Storage 4.8 in external mode]
 
 |Deploying and managing Red Hat OpenShift Container Storage on existing Google Cloud clusters
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_and_managing_openshift_container_storage_using_google_cloud/[Deploying and managing OpenShift Container Storage 4.7 using Google Cloud]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_and_managing_openshift_container_storage_using_google_cloud/[Deploying and managing OpenShift Container Storage 4.8 using Google Cloud]
 
 |Deploying Red Hat OpenShift Container Storage to use local storage on IBM Z infrastructure
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_openshift_container_storage_using_ibm_z_infrastructure/[Deploying OpenShift Container Storage using IBM Z]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_openshift_container_storage_using_ibm_z_infrastructure/[Deploying OpenShift Container Storage using IBM Z]
 
 |Deploying Red Hat OpenShift Container Storage on IBM Power Systems
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_openshift_container_storage_using_ibm_power_systems/[Deploying OpenShift Container Storage using IBM Power Systems]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_openshift_container_storage_using_ibm_power_systems/[Deploying OpenShift Container Storage using IBM Power Systems]
 
 |Deploying Red Hat OpenShift Container Storage on IBM Cloud
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_openshift_container_storage_using_ibm_cloud/[Deploying OpenShift Container Storage using IBM Cloud]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_openshift_container_storage_using_ibm_cloud/[Deploying OpenShift Container Storage using IBM Cloud]
```

Comment:
This link is a 404, looks like the product name was changed in title here:

https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_openshift_data_foundation_using_ibm_cloud/index

Reply:
acked